packages/x-date-pickers/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@mui/x-date-pickers",
-  "version": "7.0.0-alpha.3",
+  "version": "7.0.0-alpha.4",
```

Comment:
Version bump due to `@mui/base` upgrade in https://github.com/mui/mui-x/commit/200c7f7337d6220a65089b97ac2452e86b978de0#diff-e58865fd90960990fbfab6c723e16a26839669aebd39fafbefac5fc641889e49